lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/StateMachineInitializer.java

```diff
@@ -19,10 +20,13 @@
 import java.util.Map;
 
 import static com.fasterxml.jackson.core.JsonParser.Feature.STRICT_DUPLICATE_DETECTION;
+import static com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES;
 
 public class StateMachineInitializer {
     private static final ObjectMapper yamlOm =
-            new ObjectMapper(new YAMLFactory()).configure(STRICT_DUPLICATE_DETECTION, true);
+            new ObjectMapper(new YAMLFactory())
+                    .configure(STRICT_DUPLICATE_DETECTION, true)
+                    .configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Are you sure you want this? I think it's good if we fail fast if the YAML is badly formatted...

If you want to scope this more tightly, you can add `@JsonIgnoreProperties(ignoreUnknown = true)` to the class that you want to allow additional properties. (Or just add the properties!)

Reply:
I didn't think you could use that on a record, but I was wrong :)